Jenkinsfile

```diff
@@ -1,77 +1,89 @@
-//configuration
 properties([[$class: 'jenkins.model.BuildDiscarderProperty', strategy:
-			[$class: 'LogRotator', numToKeepStr: '100', artifactNumToKeepStr: '20']
-			]])
-			
-
-node {
-    echo "current branch: ${env.BRANCH_NAME}"
-}
-
+            [$class: 'LogRotator', numToKeepStr: '100', artifactNumToKeepStr: '20']
+            ]])
+            
 pipeline {
     agent any
+
     stages {
-        stage('Setup') {
+        stage('Build') {
             steps {
-                
-                sh "git submodule init" 
-
-                sh "git submodule update --init --recursive"
```

Comment:
I'm surprised this is no longer needed.

Reply:
Because of how the "aionnetwork" job Jenkins job is set up, the "Configure" screen of the Web UI is where github/SCM-related stuff is configured (i.e. repo name, branch, etc).  In that screen, there is a box you can check to tell it to pull recursively.  I switched that on.  I figured it's better that way, instead of spreading the behaviour of github/SCM around in multiple places.

Reply:
ok, sounds good